packages/core/chrome/core-chrome-browser/src/contracts.ts

```diff
@@ -51,6 +51,8 @@ export interface ChromeStart {
   recentlyAccessed: ChromeRecentlyAccessed;
   /** {@inheritdoc ChromeDocTitle} */
   docTitle: ChromeDocTitle;
+  /**  */
+  customBranding: CustomBranding;
```

Comment:
I don't think we should be changing chrome interface here.

Reply:
Ok thanks!! I am refactoring